### DISCUSSION TITLE
I have a problem with the next.js blog

### DISCUSSION THREAD

**Discussion by liuliuliucome** [vercel/next.js#36932] [Help]
My blog uses the library of next-i i18next, and the CMS system is a service of strapi.
I used the default en during development, and the fr language

Under en language, everything works fine. When I view the blog list step by step from fr/blog and then select a post to view the details there is no problem, for example fr/blog/first-post
If I copy the URL and reopen it, or refresh the browser, I get an error:
TypeError: Cannot read properties of undefined (reading 'attributes')

Probably I'm not getting the API data, I use getStaticPaths and getStaticProps function when requesting the API and use axios.get to request the API

**Comment by icyJoseph** [comment]
Hi,

Most likely the issue comes from `fallback` pages.

For example this component:

```tsx
export const PokeView = ({ pokemon }: PokeViewProps) => {
  const router = useRouter();

  if (router.isFallback) return <div>Loading...</div>;

  return (
    <>
      <Head>
        <title>{pokemon.name} | Poké Adventure</title>
      </Head>

      <PokeCard pokemon={pokemon} />
    </>
  );
};
```

If I don't use `router.isFallback`, to render a fallback UI, `pokemon` would be undefined. This page uses `getStaticProps`, and `getStaticPath`.

Could you make sure that you are handling the `router.isFallback` case in your application?

Otherwise, if you have a repository or a code sample to look at, I'd be glad to help out.

**Reply by liuliuliucome** [reply]
Hi friend, thanks for your help. 
I added the fallback processing using your suggestion and it successfully solved the problem I was having. 
Thank you very much for your advice!

**Reply by icyJoseph** [reply]
No worries. Please do mark as answered :)
